tools/boottool.sh

```diff
@@ -46,19 +51,40 @@ processOptions()
   echo "Repo=$REPO"
 }
 
+getLatestReleaseName()
+{
+  latestRepoUrl="https://api.github.com/repos/$OWNERNAME/$REPO/releases/latest"
+  echo $latestRepoUrl
+  
+  response=$(curl -sw "%{http_code}" \
+              -H "Accept: application/vnd.github+json" \
+              -H "Authorization: Bearer $GITHUB_OAUTH_TOKEN" \
+              $latestRepoUrl ) 
+
+  http_code=$(tail -n1 <<< "$response")
+  if [ "$http_code" == "$CURL_REPO_SUCCESS" ]; then
+    newReleaseData=$(sed '$ d' <<< "$response") 
+    
+    repo_json=`jq '.' <<< "$newReleaseData"`
+    RELEASENAME=$(echo $repo_json | jq -r ".name")
+    echo "Latest release name=$RELEASENAME"
+  fi
+}
+
 #process the options passed by user
 processOptions $*
 
 #Print Syntax
 printSyntax()
 {
   echo "Pass repo/release name -- usage: boottool.sh --repo <repo> --release <releasename> --uname [ownername]" 
+  echo "If --release is not passed, it will use the latest release"
   exit 1;
 }
 
 [ -z "${OWNERNAME}" ] && OWNERNAME=$DEFAULT_OWNER && echo "Using default owner name ZOSOpenTools";
 [ -z "${REPO}" ] && (printSyntax)
-[ -z "${RELEASENAME}" ] && printSyntax
+[ -z "${RELEASENAME}" ] && getLatestReleaseName
```

Comment:
A question, what is the behavior if latest release fetching fails?

Reply:
Added another check if RELEASENAME is not updated by getLatestReleaseName